src/test/test-sleep-util.c

```diff
@@ -117,33 +117,6 @@ TEST(sleep) {
         log_info("Suspend-then-Hibernate configured and possible: %s", r >= 0 ? yes_no(r) : STRERROR(r));
 }
 
-TEST(fetch_batteries_capacity_by_name) {
```

Comment:
Hmm, I see the commit message, but if this is just a move of the code with no functional change, I'd prefer to keep the test.

Reply:
After moving the functions into `sleep/`, introducing a test becomes non-trivial.

I'd like to introduce `libsleep`, i.e. move the whole `sleep-util` into `sleep/`, and link logind to it. WDYT?